Vagrantfile

```diff
@@ -18,17 +18,90 @@ end
 ENV['ANSIBLE_CONFIG'] = ansible_config_file
 
 NMONS        = ENV["VMS"] || settings['vms']
-SUBNET       = settings['subnet']
 BOX          = settings['vagrant_box']
 BOX_VERSION  = settings['box_version']
 memory       = settings['memory']
 
+SUBNET_PREFIX          = settings['subnet_prefix']
+SUBNET_ASSIGNMENT_DIR  = "/tmp/volplugin_vagrant_subnets/"
+SUBNET_ASSIGNMENT_FILE = "subnet_assignment.state"
```

Comment:
Good catch.  Thanks!

I'm curious:  what other directories would it be created in?  Do we invoke vagrant from some other CWD sometimes?